frontend/src/javascript/stage/Painter.js

```diff
@@ -61,4 +61,28 @@ export class Painter {
     getLayer() {
         return this.layers[this.layerPosition];
     }
-}
+
+    rescalePoints(points) {
+        let minX = Math.min.apply(null, points.map(function(point) { return point.x; }));
+        let maxX = Math.max.apply(null, points.map(function(point) { return point.x; }));
+        let minY = Math.min.apply(null, points.map(function(point) { return point.y; }));
+        let maxY = Math.max.apply(null, points.map(function(point) { return point.y; }));
+
+        let scaleX = 1;
+        let scaleY = 1;
+        let centerX = minX += (maxX - minX) / 2
+        let centerY = minY += (maxY - minY) / 2
+
+        if (maxX - minX != 0)
+            scaleX = 240 / (maxX - minX);
```

Comment:
Patrząc na to, totalnie nie wiem, skąd wzięło się 240. Dlaczego nie 420? Dałbym const xxx = 240; przed tym, gdzie xxx to wytłumaczenie, czym jest ta wartość. Ew. pobrał ją z innego miejsca, jeśli ma sens (np. wielkość canvasa)